DOCS-2275 Lint README (#711)
DataDog/puppet-datadog-agent: README.md

### Requirements

> The Datadog Puppet module supports Linux and Windows and is compatible with Puppet >= 4.6.x or Puppet Enterprise version >= 2016.4. For detailed information on compatibility, check the module page on [Puppet Forge][1].

Reviewer: the link text was more accurate before. The link takes you to the Agent module page on the Forge, not just to the Forge.

Reviewer: ```suggestion
The Datadog Puppet module supports Linux and Windows and is compatible with Puppet >= 4.6.x or Puppet Enterprise version >= 2016.4. For detailed information on compatibility, check the [module page on Puppet Forge][1].
```

To enable reporting of Puppet runs to your Datadog timeline, enable the report processor on your Puppet master and reporting for your clients. The clients send a run report after each check-in back to the master.

> 1. Install the [dogapi][3] gem on your system. Restart puppetserver after the gem has been installed.

Reviewer: ```suggestion
1. Install the [dogapi][3] gem on your system. Restart puppetserver after the gem is installed.
```
